pyvocz/views.py

```diff
@@ -320,7 +319,7 @@ def event(series_slug, date_slug):
                                 date_slug=proper_date_slug))
 
     github_link = "https://github.com/pyvec/pyvo-data/blob/master/{filepath}" \
-            .format(filepath=event._source)
+        .format(filepath=event._source)
```

Comment:
According to PEP 8, the [preferred way of wrapping long lines](https://www.python.org/dev/peps/pep-0008/#maximum-line-length) is parentheses, not backslash. Please use that. (And teach it to the tool, if possible.)